tslint.json

```diff
@@ -1,31 +0,0 @@
-{
-  "extends": ["tslint:latest", "tslint-sonarts", "codelyzer", "tslint-config-prettier", "./packages/stark-build/config/tslint.json"],
```

Comment:
Question... why was this `tslint.json` file removed? Is not needed anymore? I guess IntelliJ needed it right?

Reply:
Actually, it was a useless file since we have a tslint.json file in every package/application we have in Stark project